org.damask_multiphysics.DAMASK_grid.yaml

```diff
@@ -0,0 +1,68 @@
+id: org.damask_multiphysics.DAMASK_grid
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: DAMASK_grid
+finish-args:
+  - --filesystem=xdg-documents
+
+modules:
+  - name: grid_petsc
+    buildsystem: simple
+    build-options:
+      env:
+        PETSC_ARCH: "gfortran-openmpi"
+        PETSC_DIR: "/run/build/grid_petsc"
+    build-commands:
+      - ./configure --with-shared-libraries=1 --with-mpi-f90module-visibility=0
```

Comment:
this should be `buildsystem: autotools`

Reply:
This was not resolved.

Reply:
If you are using autotools, you do not need to run `./configure` that's run by default.

Add the extra args as `config-opts`

Reply:
no I understand, thanks!